### ISSUE TITLE
Wireframe models

### ISSUE THREAD

**Issue by Doprez** [Nicogo1705/Stride.BepuPhysics#12]
It would be a huge help debugging if there were Wireframe models. Below are places where Stride is using Bullet models to create them at design time:
- https://github.com/stride3d/stride/blob/938da4928cbddd9194f9a07c7380d5cc115f521e/sources/engine/Stride.Physics/Engine/PhysicsShapesRenderingService.cs#L15
- https://github.com/stride3d/stride/blob/938da4928cbddd9194f9a07c7380d5cc115f521e/sources/engine/Stride.Physics/Engine/PhysicsProcessor.cs#L35

As far as I know we can access each Bepu collider and read the vertices (at least for convex hulls) so hopefully this will be an easy conversion but this was a very surface level look.

**Comment by Nicogo1705** [comment]
Just an other possible way : 

https://github.com/stride3d/stride/pull/517

**Comment by Doprez** [comment]
https://github.com/Doprez/Stride.DebugShapes/tree/main Updated Profans PR into a seperate repo rather than waiting on PR's in the main Stride repo. It works pretty well from what I can tell and we can use the line draw to draw convex shapes until a better solution is made.

**Comment by Doprez** [comment]
https://github.com/jeske/StrideWireframeShader another option for a wireframe shader

**Comment by Doprez** [comment]
https://github.com/Nicogo1705/Stride.BepuPhysics/pull/32 I dont know if I will spend too much time on this but I started what I think could be useful for this.

**Comment by Doprez** [comment]
@Nicogo1705 I guess we can think about closing this now, unless you think we should consider design time wireframes as well. We could make it a new issue to make these not remain stagnant.
